keyvi/include/keyvi/dictionary/fsa/internal/value_store_persistence.h

```diff
@@ -136,6 +135,53 @@ struct RawPointerForCompare final {
   HashCodeTypeT hashcode_;
 };
 
+/**
+ * Comparison for the string value store which does not use length prefixes
+ */
+template <class PersistenceT, class HashCodeTypeT = int32_t>
+struct RawPointerForCompareString final {
```

Comment:
So this will be removed once current `StringValueStore` is deprecated, right ?

Reply:
Yes, we have to keep the reader for some time, but the writing part can be removed as part of the reimplementation.